Packs/CommunityCommonScripts/Scripts/BulkCloseIncidents/BulkCloseIncidents.py

```diff
@@ -0,0 +1,54 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+def send_post_request(endpoint, data=None):
+    try:
+        if data:
+            data = json.dumps(data)
+            res = execute_command("core-api-post", {
+                "uri": endpoint,
+                "body": data
+            })
+    except requests.HTTPError as e:
+        return_error(f"HTTP error: {e}")
+    return res
+
+
+def main():
+    args = demisto.args()
+    query = args.get('query')
+    try:
+        batch_size = int(args.get('batch_size'))
+    except ValueError:
+        return_error("'batch_size' must be integer.")
+    try:
+        # only get active incidents (status == 1)
+        incidents = [inc['id'] for inc in execute_command('getIncidents', {
+                                'query': query,
+                                'size': '10000'
+                            })['data'] if inc['status'] == 1]
+    except:
```

Comment:
Using a bare `except` is discouraged because it catches all exceptions, including system-level ones like `SystemExit` and `KeyboardInterrupt`, which can mask issues and make debugging difficult. The current error message on line 31 is also misleading; an exception here indicates a failure to fetch incidents, not that no incidents were found.